app/(main)/markets/AiChat.tsx

```diff
@@ -144,22 +198,21 @@ export const AiChat = ({ id }: AiChatProps) => {
   );
 };
 
-interface MessageProps {
-  children: React.ReactNode;
-  role: Role;
+interface MessageProps extends PropsWithChildren {
+  role: string;
 }
 
 const Message = ({ children, role }: MessageProps) => {
-  const isSystem = role === 'system';
+  const isSystem = role === 'assistant';
```

Comment:
For consistency sake
```suggestion
  const isAssistant = role === 'assistant';
```